docs/modules/community/pages/mailing-lists.adoc

```diff
@@ -0,0 +1,165 @@
+specific questions is generally directly proportional to the amount of effort you spend formulating your question.
+Eric Raymond and Rick Moen have even written an essay entitled "Asking Smart Questions" precisely on this topic.
+Although somewhat militant, it is definitely worth reading.
+
+*Note*: Please do NOT send your Java problems to the two authors. They welcome feedback on the FAQ's contents, but are
+simply not a Java help resource. Follow the essay's advice and choose your forum carefully.
+
+*Keep your email short and to the point*
+
+If your email is more than about a page of text, chances are that it won't get read by very many people. It is much
+better to try to pack a lot of informative information (see above about asking smart questions) into as small of an
+email as possible. If you are replying to a previous email only quote the parts that you are replying to and to remove
+the unnecessary bits. This makes it easier for people to follow a thread as well as making the email archives easier
+to search and read.
+
+*Do your best to ensure that you are not sending HTML or "Stylelized" email to the list*
+
+If you are using Outlook or Outlook Express or Eudora, chances are that you are sending HTML email by default. There is
+usually a setting that will allow you to send "Plain Text" email. If you are using Microsoft products to send email,
+there are several bugs in the software that prevent you from turning off the sending of HTML email.
+
+*Watch where you are sending email*
+
+The majority of our mailing lists have set the Reply-To to go back to the list. That means that when you Reply to a
+message, it will go to the list and not to the original author directly. The reason is because it helps facilitate
+discussion on the list for everyone to benefit from. Be careful of this as sometimes you may intend to reply to a
+message directly to someone instead of the entire list. The appropriate contents of the Reply-To header is an age-old
+debate that should not be brought up on the mailing lists. You can examine opposing points of view condemning our
+convention and condoning it. Bringing this up for debate on a mailing list will add nothing new and is considered off-topic.
+
+*Do not cross post messages*
+
+In other words, pick one mailing list and send your messages to that mailing list only. Do not send your messages to
+multiple mailing lists. The reason is that people may be subscribed to one list and not to the other. Therefore, some
+people will only see part of the conversation.
+
+== James mailing lists
+
+=== Server user list
+
+*Low traffic*
+mailto:server-user-subscribe@james.apache.org[Subscribe]
+mailto:server-user-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/server-user@james.apache.org[Archive]
+
+This is the list where users of the Apache James (Server) meet and discuss issues. Developers are also expected to be
+subscribed to this list to offer support to users of Apache James (Server).
```

Comment:
I think we could benefit from use cases:

 - I want to receive Apache James announcements
 - I run into xx problem and want some help. In which case I should search the mail archive first.